src/ai/backend/manager/models/session.py

```diff
@@ -1216,49 +1216,48 @@ async def resolve_abusing_reports(
         return [(await con.resolve_abusing_report(info, self.access_key)) for con in containers]
 
     _queryfilter_fieldspec = {
-        "session_type": ("session_type", lambda s: SessionTypes[s]),
-        "name": ("name", None),
-        "image": ("image", None),
-        "architecture": ("architecture", None),
-        "domain_name": ("domain_name", None),
-        "group_name": ("groups_group_name", None),
+        "id": ("sessions_id", None),
+        "session_type": ("sessions_session_type", lambda s: SessionTypes[s]),
```

Comment:
I wonder if the prefix "session_" is necessary since it is not aligned with the others.
```suggestion
        "type": ("sessions_session_type", lambda s: SessionTypes[s]),
```